docs/compose-file.md

```diff
@@ -24,6 +24,64 @@ As with `docker run`, options specified in the Dockerfile (e.g., `CMD`,
 `EXPOSE`, `VOLUME`, `ENV`) are respected by default - you don't need to
 specify them again in `docker-compose.yml`.
 
+## Versioning
+
+It is possible to use different versions of the `compose.yml` format.
+Below are the formats currently supported by compose.
```

Comment:
I think we might want to start by only talking about the latest version of the config up front, with a section at the bottom which covers v1 (and maybe a link to the V1 section from here).

We'll have to include some notes about how to migrate to the v2 config in the section at the bottom.

Does that sound right @moxiegirl ?